jgrapht-core/src/main/java/org/jgrapht/alg/shortestpath/BaseBidirectionalShortestPathAlgorithm.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * (C) Copyright 2019-2019, by Semen Chudakov and Contributors.
```

Comment:
This code was copied pasted from another file and you changed authorship. Please do not do that.

Reply:
Still needs to change to the original author. Refactoring techniques such as "extract supertype" do not change the author of the code.

Reply:
Done.